src/tsdb.c

```diff
@@ -143,7 +143,7 @@ Series *NewSeries(RedisModuleString *keyName, CreateCtx *cCtx) {
     }
 
     if (!cCtx->skipChunkCreation) {
-        Chunk_t *newChunk = newSeries->funcs->NewChunk(newSeries->chunkSizeBytes);
+        Chunk_t *newChunk = newSeries->funcs->NewChunk(newSeries->chunkSizeBytes, newSeries->chunkSizeBytes);
```

Comment:
@slice4e I think we should use here newSeries->chunkSizeBytes/2 cause the client requested the chunk to be that size.
Also following our conversation we need to think using dynamically growing array here.